app/admin/tokens.rb

```diff
@@ -0,0 +1,13 @@
+ActiveAdmin.register Token do
+  permit_params :name, :ticker, :hard_cap, :precision, :user_id
+  form do |f|
+    inputs do
+      input :user, label: "Owner", collection: User.all.map{|u| ["#{u.email}", u.id]}
```

Comment:
"#{u.email}" - you have created a string just to display variable in it. `u.email` is sufficient